packages/libs/web-common/src/user-dataset-upload-config.tsx

```diff
@@ -45,18 +45,22 @@ export const uploadTypeConfig: DatasetUploadTypeConfig<ImplementedUploadTypes> =
         },
         renderInfo: () => (
           <p className="formInfo">
-            Complete the Upload My Normalized RNA-Seq form. Name, Summary and
-            Upload File/URL are required. Three types of files – bigWig, counts
-            and index – are supported in the uploaded zip file. bigWig files are
-            not required but will allow visualization in the genome browser if
-            included. The counts (FPKM/TPM) files should be tab-delimited and
+            Upload your Normalized RNA-Seq in a tar.gz, tgz or zip file. Three
+            types of files – bigWig, counts and index – are supported in the
+            uploaded zip file.
```

Comment:
change to

Upload your Normalized RNA-Seq dataset files by zipping them into a tar.gz, tgz or zip file. Three
            types of files – bigWig, counts and index – are supported in the
            uploaded zipped file.